src/windows/grab.rs

```diff
@@ -44,15 +45,18 @@ impl From<HookError> for GrabError {
     }
 }
 
-pub fn grab<T>(callback: T) -> Result<(), GrabError>
+pub fn grab<T>(event_types: EventTypes, callback: T) -> Result<(), GrabError>
 where
     T: FnMut(Event) -> Option<Event> + 'static,
 {
     unsafe {
         GLOBAL_CALLBACK = Some(Box::new(callback));
-        set_key_hook(raw_callback)?;
-        set_mouse_hook(raw_callback)?;
-
+        if event_types.keyboard {
+            set_key_hook(raw_callback)?;
```

Comment:
Btw, the current keyboard hook can break things.
With my (very custom) keyboard, a few shortcuts do not work if I register this hook.